zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedNodepoolChecker.scala

```diff
@@ -0,0 +1,50 @@
+package com.broadinstitute.dsp
+package zombieMonitor
+
+import cats.effect.{Concurrent, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import fs2.Stream
+import io.chrisdavenport.log4cats.Logger
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+/**
+ * Scans through all active nodepools and check if they still exist in Google.
+ * If they're deleted from Google, we're updating them to `DELETED` in leonardo DB.
+ * We're also updating any kubernetes apps running on these nodepools as `DELETED`.
+ */
+object DeletedNodepoolChecker {
```

Comment:
Should the name reflect the fact that errored nodepools are being checked as well?